src/views/error-not-found.njk

```diff
@@ -0,0 +1,62 @@
+{% extends "layout.njk" %}
+
+{% from 'govuk/components/details/macro.njk' import govukDetails %}
+
+{% block pageTitle %}
+  Appeal a late filing penalty
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">
+        Page not found
+      </h1>
+      <p>If you typed the web address, check it is spelled correctly.</p>
+      <p>If you pasted the web address, check you copied the entire address.</p>
+      <p>If the web address is correct or you selected a link, the page may have moved.
+        <a href="/appeal-a-penalty">Go to the Appeal a late filing penalty start page</a>.
```

Comment:
could we reference the root path here? in case this was to change

Reply:
can you also double check this link works? strangely (could be my environment) it's not working for me

Reply:
again, was working on vagrant, but not on docker. Will have to check it out

Reply:
fixed